.changes/1.35.8.json

```diff
@@ -0,0 +1,7 @@
+[
```

Comment:
This file should not be touched. You'll need to remove this from your commit. You can do this with some variation of:

```
git reset HEAD^ -- path/to/file
git commit --amend
```